CMSgov/onemac
# OY2-16976: Base Waiver: Verbiage and Validation for Submission

Story: https://qmacbis.atlassian.net/browse/OY2-16976
Endpoint: https://d2ajx5kog67s2l.cloudfront.net/

### Details
Updating forms for package view means they should be accessed from and return to the package dashboard.

### Changes
- returned the legacy New Submission functionality to previous incarnation
- created new triage choice pages for the new form versions

### Test Plan
1. Login as a state submitter
2. Navigate to Package Dashboard
3. Click "New Submission" button
4. Click "Waiver" then "Base Waiver"
5. Submit a Base Waiver using the form
6. after clicking submit, notice that you are returned to the package dashboard and are shown the "Submission success" message
